src/middleware.ts

```diff
@@ -13,7 +13,7 @@ export function middleware(request: NextRequest) {
     default-src 'self';
     script-src 'self' 'nonce-${nonce}' 'strict-dynamic' https: 'unsafe-inline' 'unsafe-eval';
     style-src 'self' https: 'unsafe-inline';
-    img-src 'self' https://stellar.creit.tech/wallet-icons/ blob: data:;
+    img-src 'self' https://id.lobstr.co/ https://stellar.creit.tech/wallet-icons/ blob: data:;
```

Comment:
Do we still need this?

Reply:
Let me check

Reply:
we do not, because the Account component generates data urls. :)